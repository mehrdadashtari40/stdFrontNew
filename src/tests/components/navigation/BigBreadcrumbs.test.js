import React from 'react';
import { shallow } from 'enzyme';
import { BigBreadcrumbs } from '../../../common';

test('Make Snapshot file , BigBreadcrumbs Component', () => {
	const wrapper = shallow(<BigBreadcrumbs />);

	expect(wrapper).toMatchSnapshot();
});
