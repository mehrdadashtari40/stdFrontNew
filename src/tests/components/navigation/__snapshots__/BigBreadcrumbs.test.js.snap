// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make Snapshot file , BigBreadcrumbs Component 1`] = `
<div
  className="undefined big-breadcrumbs"
>
  <h1
    className="page-title txt-color-blueDark"
  >
    <i
      className="fa fa-fw fa-home"
    />
     undefined
  </h1>
</div>
`;
